.stylelintignoe

```diff
@@ -0,0 +1,16 @@
+app.fork5/**
```

Comment:
typo in filename `.stylelintigno**r**e`

Reply:
fixed 1f62fa2daa07598c77c0af0d1c53c5e452f087f8